errors/graph.go

```diff
@@ -0,0 +1,412 @@
+package errors
+
+// This file contains all of the errors that can be generated from the
+// docker/graph component.
+
+import (
+	"net/http"
+
+	"github.com/docker/distribution/registry/api/errcode"
+)
+
+var (
+	// ErrorCodeNoSuchImage is generated when we can't find the image.
+	//
+	ErrorCodeNoSuchImage = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "NOSUCHIMAGE",
+		Message:        "could not find image: %v",
+		HTTPStatusCode: http.StatusNotFound,
+	})
+
+	// ErrorCodeImageIDConflict is generated when the expected id does not
+	// match the id of the loaded image.
+	ErrorCodeImageIDConflict = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "IMAGEWITHWRONGIMAGEID",
```

Comment:
Pleas make sure the Value field is the same as the ErrorCode variable, just without the "ErrorCode" part and the rest of it uppercase. e.g.  `IMAGEIDCONFLICT` in this case.